nuopc.runseq

```diff
@@ -1,5 +1,5 @@
 runSeq:: 
-@1800
+@3600
```

Comment:
I don't understand why this change is here after rebasing? Would you like me to rebase this branch on `1deg_jra55do_ryf`?